src/gasman.c

```diff
@@ -2302,8 +2331,8 @@ UInt CollectBags (
 
     GAP_ASSERT(SanityCheckGasmanPointers());
 
-    /* return success                                                      */
-    return 1;
+    /* success                                                      */
```

Comment:
Perhaps you meant to remove the word "success"? :-). In any case, I'd just remove the whole pointless comment
```suggestion
```

Reply:
I think I just somehow broke the comment